src/invite/Invite.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
+import { noop } from 'lodash'
+import React from 'react'
+import { Share } from 'react-native'
+import { Provider } from 'react-redux'
+import { mocked } from 'ts-jest/utils'
+
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { createMockStore } from 'test/utils'
+
+import Invite from './Invite'
+import { createDynamicLink } from './utils'
+
+jest.mock('./utils', () => ({
+  createDynamicLink: jest.fn(),
+}))
+
+const { press } = fireEvent
+const mockedCreateDynamicLink = mocked(createDynamicLink)
+
+describe('Invite', () => {
+  beforeAll(() => {
+    jest.useRealTimers()
```

Comment:
do we need real timers? can we do `jest.runAllTimers` when we need instead?

Reply:
I added this for the `waitFor` call. I can try it with `runAllTimers` instead, but it will be more verbose.

Reply:
I just tried using `waitFor` with default timers, trying to progress with `runAllTimers`, but no dice. There might be something weird in the `waitFor` implementation. In theory we should only need to wait one tick on those lines, as the promise should resolve immediately once the call stack is clear, so I tried creating my own `waitOneTick` helper function, but it didn't work either. IMO the juice isn't worth the squeeze here, but I'm happy to jam on this a little longer if you feel otherwise.